### ISSUE TITLE
People: Manually assign people to photos/videos without setting a marker

### ISSUE THREAD

**Issue by joachimtingvold** [photoprism/photoprism#1863]
Slightly relevant to #1548 would be to add people to a video manually. I guess automatic face recognition for videos is not really feasible, but at least one should be able to add manually people that is in a video, regardless of where they appear in the video, so that they show up when filtering for that person.

**Comment by joachimtingvold** [comment]
I thought maybe this would be a duplicate of #1548 ... since a video will have a sidecar image generated (and stacked), the video seems to already have the "People" pane available.

However, manual tagging of said video image would probably pose some problems? I.e. you risk having to manually add more persons to an image than is actually in the image (which could cause problems with future matching of same person).

So maybe this needs to be generalized to "be able to add people to an image without them being part of said image". This would also apply for scenarios where you maybe want to add people to a photo that's behind the camera...

**Comment by graciousgrey** [comment]
As you mentioned persons can already be tagged on the still image for videos. 

So the request here is to assign a person to an image/video without setting a marker on the face.

**Comment by Szeraax** [comment]
Oh, this is the issue that I should have upvoted. Darnit. I'd pay $20 to fix a bug if that meant that this gets completed sooner :)

**Comment by Szeraax** [comment]
Dunno about videos, but at least for images, if you add people to the IPTC Xmp.iptcExt.PersonInImage field, they will show up in PP in the subject field. You can use tools like exiv2 to set that field. I view all fields with exiv2 via: 

```
exiv2.exe -p a pr path\to\image.jpg
```

And update via: 
```
exiv2.exe -M 'add Xmp.iptcExt.PersonInImage "Roger, Carlos, Michael"' path\to\image.jpg
```

Maybe that's helpful to people who want to get tagging without waiting on it to be available in PP natively.

**Comment by KevinGelking** [comment]
Recently got the Gold membership for this :) Love the app but still really hoping for this feature and [#1548](https://github.com/photoprism/photoprism/issues/1548) .

Wouldn't it be possible to have the Face-Tagging feature use that new 'assigned people' (I'll call it 'person entry') field as it's backing?

E.g.    DB has an entry for the person entry in a picture, with an optional field for a face tag
A face tag would then create the person entry if it doens't exist and set it's tag info.
A manual person reference would still create the person entry, but without tag.

The face-AI will consider only person entries that have a face tag so it doesn't get corrupted.
"Upgrading" a manual person reference without tag to one that has a tag would be possible, too, in this scenario.

This would be immensely beneficial for cases where most of a person is visible except for the face - or if someone is wearing a costume, face paint or VR goggles.

**Comment by GlassedSilver** [comment]
> Recently got the Gold membership for this :) Love the app but still really hoping for this feature and [#1548](https://github.com/photoprism/photoprism/issues/1548) .
> 
> Wouldn't it be possible to have the Face-Tagging feature use that new 'assigned people' (I'll call it 'person entry') field as it's backing?
> 
> E.g. DB has an entry for the person entry in a picture, with an optional field for a face tag A face tag would then create the person entry if it doens't exist and set it's tag info. A manual person reference would still create the person entry, but without tag.
> 
> The face-AI will consider only person entries that have a face tag so it doesn't get corrupted. "Upgrading" a manual person reference without tag to one that has a tag would be possible, too, in this scenario.
> 
> This would be immensely beneficial for cases where most of a person is visible except for the face - or if someone is wearing a costume, face paint or VR goggles.

As someone who photographs a lot of cosplayers who would love to be able to handle such cases better - thank you for bringing up costumes and face paint.

If there was a way to tag a face but also assigning something like a character, costume or such to that - I'd be so forever grateful.

Imagine browsing your best friend's cosplay pictures and thinking - oh, who else cosplayed Hatsune Miku in my gallery?

Then you simply click on "Hatsune Miku" in "Sarah as Hatsune Miku" and a gallery view of everyone being in a Miku cosplay opens.

This would make life so much easier for so many of us. QoL heaven!

Give this idea a bounty and I'll put down some cash!
